test_files/results.sql

```diff
@@ -0,0 +1,213 @@
+--
```

Comment:
This file isn't every actually used because it's always replaced by the output of the integration test.